V0.2.0 M4

FabrizioMoggio opened the pull request:
M4 version for Fall24:

- OAS file renamed according to: https://github.com/camaraproject/CallForwardingSignal/issues/93
- Remove “-rc.n” from the version and server path: https://github.com/camaraproject/CallForwardingSignal/discussions/92
- Remove "-rc.02" from "x-camara-commonalities": https://github.com/camaraproject/CallForwardingSignal/issues/94
- updated changelog.md
- updated readme.md
- updated .feature files according to: https://github.com/camaraproject/CallForwardingSignal/issues/96
- updated Readiness CL according to: https://github.com/camaraproject/CallForwardingSignal/issues/97
- updated changelog.md according to: https://github.com/camaraproject/CallForwardingSignal/issues/98
- updated readme.md according to: https://github.com/camaraproject/CallForwardingSignal/issues/100

#### What type of PR is this?

Add one of the following kinds:
* cleanup
* documentation

#### What this PR does / why we need it:
final release for Fall24

#### Which issue(s) this PR fixes:
https://github.com/camaraproject/CallForwardingSignal/issues/94
https://github.com/camaraproject/CallForwardingSignal/issues/93
https://github.com/camaraproject/CallForwardingSignal/issues/96
https://github.com/camaraproject/CallForwardingSignal/issues/97
https://github.com/camaraproject/CallForwardingSignal/issues/98
https://github.com/camaraproject/CallForwardingSignal/issues/100

hdamker commented:
@FabrizioMoggio thanks for the release PR - have created https://github.com/camaraproject/ReleaseManagement/issues/77 for the Release Management review. 

Please update also the API Readiness checklist and the version within the .feature files as part of the PR.

hdamker commented:
@FabrizioMoggio and another important point: 

The Release Notes for a public release must be in generell a summary of all changes since the last public release, not just the delta to the last release candidate.
In your case - as there was no previous public release and everything is new - it's more a list of "What's new" as you had in the first release candidate - you don't need to mention the corrections of this new stuff which happened between the release candidates, as for someone who is taking this public release, everything is anyway new.

Note: the "full changelog" link is
`Full Changelog: https://github.com/camaraproject/CallForwardingSignal/commits/r1.3`

bigludo7 reviewed (dismissed):
LGTM
Your PR will be used as an example for me for other API Fabrizio :)

bigludo7 commented:
Correction for my comment: this @FabrizioMoggio PR will be the example once the comment from @hdamker will be fixed ;)

hdamker reviewed (changes requested):
* Please update also the API Readiness checklist and the version within the .feature files as part of the PR
* Complete the CHANGELOG for the public release with all updates, see my previous comment

hdamker commented:
BTW: release tracker updated ... r1.3 will be the public release tag, not the next pre-release.

FabrizioMoggio commented:
I’m sorry to ask for advice, I’m trying do to my best 😊 from my holidays. For M4 I should have addressed all your comments. The current PR for the CFS API is targeting the public release. What I’m not sure of is if at 30/08 the public release should be merged (if so, we missed it 😊) or if the final approval from @camaraproject/release-management_maintainers is due for M5 (09-15).

hdamker commented:
@FabrizioMoggio thank for taking car during your vacation! No worries, you have done every necessary for the M4 milestone by creating and updating this release PR so far. Indeed, the review and approval by Release Management will happen before M5 and then the merge.

@bigludo7 @chinaunicomyangfan Please review the PR and the latest changes as well and approve the PR already.

bigludo7 reviewed (dismissed):
LGTM

FabrizioMoggio reviewed (commented):
the requested changes by @hdamker have been implemented

FabrizioMoggio commented:
> * Please update also the API Readiness checklist and the version within the .feature files as part of the PR
> * Complete the CHANGELOG for the public release with all updates, see my previous comment

Dear @hdamker I don't know how to move forward. You requested a change and I have implemented it but I can't find a way to mark the requested changes as implemented. So I can't merge. Is it you that need to control and mark them as implemented?

bigludo7 reviewed (approved):
LGTM

tanjadegroot commented on documentation/API_documentation/call-forwarding-signal-API-Readiness-Checklistst.md:
```diff
@@ -1,21 +1,21 @@
 
 | Nr | API release assets  | alpha | release-candidate |  public-release<br>initial | public-release<br> stable | Status | Comments |
 |----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|:----|
 |  1 | API definition                               |   M   |         M         |    M    |    M   |   Y  | /code/API_definitions/Call_Forwarding_Signal.yaml |
```
change to new API yaml file name (call-forwarding-signal.yaml)

tanjadegroot reviewed (changes requested):
a few broken links and typos to fix and this is OK: LGTM from release mgmt

FabrizioMoggio commented:
Closed because EasyCLA problems.

Replaced by: https://github.com/camaraproject/CallForwardingSignal/pull/101
